Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Use the official Docker Hub Ubuntu 14.04 base image
+FROM ubuntu:14.04
```

Comment:
We wanna use 16.04 here, seeing 14.04 is getting rather old. Totally fine to do this in a later PR though.